src/main/java/practice/CandidateValidator.java

```diff
@@ -1,5 +1,24 @@
 package practice;
 
-public class CandidateValidator {
-    //write your code here
+import java.util.function.Predicate;
+import model.Candidate;
+
+public class CandidateValidator implements Predicate<Candidate> {
+    private static final String SEPARATOR = "-";
+    private static final int STARTINDEX = 0;
+    private static final int ENDINDEX = 1;
+    private static final int MINTERM = 10;
+    private static final int MINAGE = 35;
```

Comment:
these two "index" names does not really give us more context, so let's think bout some other name to make it more understandable, like what do we get by those indexes? 
Also the constant naming, words are divided by "_"
```suggestion
    private static final int START_INDEX = 0;
    private static final int END_INDEX = 1;
    private static final int MIN_TERM = 10;
    private static final int MIN_AGE = 35;
```